### PR TITLE
Modified .html , .CSS and README.md files

### PR DESCRIPTION
**Pull request by Tshepo45**
-Edit project description
-Modify property value on .CSS file
-Organized .html file professionally
-Coding partners committed on the contact form master branch

### CODE REVIEW

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Tshepo45 , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

## Required Changes ♻️
_Check the comments under the review._
- [ ] You have implemented the form quite well but it is horizontally scrollable, kindly fix this issue.
srcoll
- [ ] Please add the different button states to your contact form. This is part of the project requirement. Good-looking projects can easily land you a job.
**design**
states1

states

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [OPTIONAL] Since this file is not used in this project, you can delete it.
murple

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -1288,6 +1292,8 @@ ul {
 .btn:active {
```
Comment (souad988) [inline styles.css:1292]: Kindly follow figma design colors and effects as requested by the previous reviewer as your button doesn't change color when we hover  over it also when pressed the color is not as it is in figma, you have done a great work in this project and you need just to stick to your template design as the goal in this activity is to replicate the given design 😊

temp1 button effects

File: index.html [hunk 2]
```diff
@@ -212,12 +212,8 @@ <h2 class="About" >About <br> Myself </h2>
  <section class="form">
```
Comment (souad988) [inline index.html:212]: Great work so far @Tshepo45 , I have just to mention that horizontal scrolling mentioned by the previous reviewer is note solved , I noticed that you are using absolute positioning  to align your sections and this is not correct as it is used only if you want to remove a block from the natural flow of the dom and make it overlap other components which is not the case here. Please make sure to solve this problem before requesting the next review

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Tshepo45 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865

## Required Changes ♻️
_Check the comments under the review._
_Also feel free to ask is you have any question 😊

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -1291,7 +1291,7 @@ h1 {
 
 .btn:active {
   opacity: 1;
-  color: #5E6C84;
+  color: #5e6c84;
 }
```
Comment (ABDULALI3468) [inline styles.css:1295]: Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file. Currently, your styles don't match the Figma design.

## Solution:--

```
.btn:hover{
    background-color: #6070FF;
    color: white;
}
```

```
.btn:active{
    background-color: #2230D2;
    color: white;
}
```

## Demonstration:--

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @Tshepo45 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -1285,13 +1285,13 @@ h1 {
...
   background-color: #6070ff;
+  color: white;
 }
 
 .btn:active {
-  opacity: 1;
-  color: #5E6C84;
+  background-color: #2230d2;
+  color: white;
 }
```
Comment (ABDULALI3468) [inline styles.css:1295]: [OPTIONAL] Please consider removing the border from the button when it is being `hovered or clicked` to make it in accordance with the design. Moreover consider reviewing the width and height of the button also.

## Solution:-

```
.btn:hover, .btn:active {
  border: none;
}
```

```
.btn {
    padding: 12px 16px;
    width: 142px;
    height: 48px;
    border-radius: 8px;
}
```

**Review by ABDULALI3468 — APPROVED** [review]
Hi @Tshepo45 , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions but necessary for improvement

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**